absl/testing/absltest.py

```diff
@@ -1223,10 +1223,15 @@ def _sorted_list_difference(expected, actual):
 
 def _walk_structure_for_problems(a, b, aname, bname, problem_list):
   """The recursive comparison behind assertSameStructure."""
```

Comment:
Since, we're here, go ahead and update the docs to indicate that mixed mapping type are supported, but the order of keys is not considered when comparing them.